pythonkr_backend/pythonkr_backend/celery.py

```
@@ -0,0 +1,41 @@
+
+# Using a string here means the worker doesn't have to serialize
+# the configuration object to child processes.
+# - namespace='CELERY' means all celery-related configuration keys
+#   should have a `CELERY_` prefix.
+app.config_from_object('django.conf:settings', namespace='CELERY')
+
+# Load task modules from all registered Django apps.
+app.autodiscover_tasks()
+
+
+@worker_init.connect()  
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)